corehq/apps/app_manager/templates/app_manager/v2/form_view.html

```diff
@@ -8,7 +8,7 @@
 {% endblock %}
 
 {% block case_management_js %}{{ block.super }}
-  {% if form.uses_cases %}
+  {% if form.source %}
```

Comment:
@biyeun `master` uses `form.source` but you changed it in https://github.com/dimagi/commcare-hq/commit/c8403b6d074170dea37ebc132b866a370cf45b0f - do you remember why?